tests/ssl-opt.sh

```diff
@@ -6616,11 +6616,12 @@ run_test    "mbedtls_ssl_get_bytes_avail: no extra data" \
             -s "Read from client: 100 bytes read$"
 
 requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
+requires_max_content_len 474
```

Comment:
Sorry if I'm missing something obvious, but why is this request size +24?

Reply:
Good question. I don't remember. I suspect I arrived at those two numbers independently and I forgot to reconcile them. I've changed those tests to be more precise and not rely on magic numbers defined elsewhere anymore.